src/routes/activityReports/handlers.js

```diff
@@ -38,7 +38,7 @@ import {
   reportApprovedNotification,
   collaboratorAssignedNotification,
 } from '../../lib/mailer';
-import { activityReportToCsvRecord, extractListOfGoalsAndObjectives, deduplicateObjectivesWithoutGoals } from '../../lib/transform';
```

Comment:
we should remove this function if it's not in use anywhere, that way we aren't on the hook for testing it